draft-ietf-wimse-arch.md

```diff
@@ -79,8 +79,20 @@ Attestation is the function through which a task verifies the identity of a sepa
 
 ## Workload Identity
 
+A workload identity may be composed of many attributes describing different aspects of a workload. These attributes may describe different sets of information such as a workload's role within a system, the software that the workload runs or the hardware that the workload runs on. Sets of attributes may be determined by different authorities running in different parts of the system. WIMSE defines a **Workload Identifier** to used in representing and referencing the workload.  It allows different authorities of sets identity attributes to reference the workload in a consistent way.
```

Comment:
Typo in "to used in representing". Maybe "to be used in representing"?
Typo in "different authorities of sets"... Maybe "different authorities to set"? The full sentence may revise.